core/amber/src/test/scala/edu/uci/ics/texera/workflow/common/tuple/TupleSpec.scala

```diff
@@ -97,4 +97,16 @@ class TupleSpec extends AnyFlatSpec {
     assert(line == tuple2json(newTuple))
 
   }
+
+  it should "calculate hash" in {
+    val inputSchema =
+      Schema.newBuilder().add(integerAttribute).add(stringAttribute).add(boolAttribute).build()
+    val inputTuple = Tuple
+      .newBuilder(inputSchema)
+      .add(integerAttribute, 1)
+      .add(stringAttribute, "string-attr")
+      .add(boolAttribute, true)
+      .build()
+    println(s"hash ${inputTuple.hashCode()}")
+  }
```

Comment:
I think it's better to add more test cases, especially for double and timestamp types. We need to make sure both Java and Python return the same result on `doubleToLongBits` and `getTime`.

Reply:
Added tests for all types.